base_types/kotlin/VectorAnalyticsEvent.kt

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2021 New Vector Ltd
```

Comment:
I believe you have to use the matrix.org copyright in this project (applicable to all added files)

Reply:
I applied the same header as the classes in [types/kotlin2](https://github.com/matrix-org/matrix-analytics-events/tree/jitpack/types/kotlin2).
I could apply this change but I would have to do the same thing for all generated classes.
WDYT